docs/sphinx/source/variables_style_rules.rst

```diff
@@ -0,0 +1,30 @@
+.. _variables_style_rules:
+
+Variables and Symbols
+=====================
+
+There is a convention on consistent variable names throughout the library:
+
+.. csv-table:: List of used Symbols and Variables
+   :file: ../../../pvlib/data/variables_style-rules.csv
+   :delim: ;
+   :header-rows: 1
+   :widths: 5, 5
+   :stub-columns: 1
+   
+For an definition and further help on symbols and units refer to the following sources:
```

Comment:
Probably need to say something about the fact that these references might not use the same terminology as us.